src/browser/service-orchestrator.ts

```
@@ -0,0 +1,274 @@
+    }
+  }
+
+  /**
+   * Create API client
+   */
+  private async createAPIClient(serviceName: string): Promise<any> {
+    // Load appropriate SDK/client
+    switch (serviceName) {
+      case 'gemini-flash':
+      case 'gemini-pro':
+        const { GoogleGenerativeAI } = await import('@google/generative-ai');
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)